### ISSUE TITLE
pxr::Tftoken global destructor can cause a freeze in C++.

### ISSUE THREAD

**Issue by MasayaNikaido** [adobe/USD-Fileformat-plugins#21]
Static TfToken  is unsafe.
These codes have direct static TfToken definitions.
- [utils/sdfMaterialUtils.cpp](https://github.com/adobe/USD-Fileformat-plugins/blob/main/utils/sdfMaterialUtils.cpp#L60)
- [fbx/src/fileFormat.cpp](https://github.com/adobe/USD-Fileformat-plugins/blob/main/fbx/src/fileFormat.cpp#L32)

I think it would be better to use the Static Token macros or pxr::Tftoken::Immortal.

#### Platform and Version
- USD 1.38.7 (monolithic build)
- Windows11 x64
- Visual Studio 2022

#### Logs taken while reproducing problem
CallStack
```
ntdll.dll!NtDelayExecution() + 20 bytes	Unknown
ntdll.dll!RtlDelayExecution() + 67 bytes	Unknown
KernelBase.dll!SwitchToThread() + 27 bytes	Unknown
usd_ms.dll!pxrInternal_v0_23__pxrReserved__::TfToken::_PossiblyDestroyRep() + 212 bytes	Unknown
fileformatUtils.dll!adobe::usd::Resolver::populateCache() + 2439 bytes	Unknown
fileformatUtils.dll!adobe::usd::Resolver::populateCache() + 5454 bytes	Unknown
fileformatUtils.dll!adobe::usd::Resolver::populateCache() + 1303 bytes	Unknown
fileformatUtils.dll!adobe::usd::Resolver::populateCache() + 1279 bytes	Unknown
fileformatUtils.dll!adobe::usd::Resolver::populateCache() + 1279 bytes	Unknown
fileformatUtils.dll!00007ffb3da8c5e2()	Unknown
ucrtbase.dll!__sys_nerr() + 691 bytes	Unknown
ucrtbase.dll!_o_free() + 222 bytes	Unknown
ucrtbase.dll!_execute_onexit_table() + 61 bytes	Unknown
fileformatUtils.dll!00007ffb3da809fd()	Unknown
fileformatUtils.dll!00007ffb3da80b22()	Unknown
ntdll.dll!RtlActivateActivationContextUnsafeFast() + 303 bytes	Unknown
ntdll.dll!LdrShutdownProcess() + 374 bytes	Unknown
ntdll.dll!RtlExitUserProcess() + 173 bytes	Unknown
kernel32.dll!ExitProcess() + 11 bytes	Unknown
```

**Comment by kwblackstone** [comment]
Thanks for bringing this to our attention, we will do an audit of all non-trivial globals.

**Comment by kwblackstone** [comment]
These specific cases are fixed in v1.0.4 with some more fixes on the way.  Please reopen if you are still having issues.
